app/notification.go

```diff
@@ -458,7 +458,39 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 		message.Add("mentions", model.ArrayToJson(mentionedUsersList))
 	}
 
-	a.Publish(message)
+	var previewedPostID string
+	if val, ok := post.GetProp(model.POST_PROPS_PREVIEWED_POST).(string); ok {
+		previewedPostID = val
+	}
+	var previewedChannel *model.Channel
+	if model.IsValidId(previewedPostID) {
+		previewedPost, err := a.GetSinglePost(previewedPostID)
+		if err != nil {
+			return nil, err
+		}
+
+		channelMembers, err := a.GetChannelMembersPage(channel.Id, 0, 10000000)
```

Comment:
~~Is this okay or should we be paging through this properly?~~

Never mind. We do this in a few other places. It looks kind of odd, but I guess we'll never have a channel with 10 million users. If we do end up with that, we definitely have bigger concerns than this breaking though.